packages/augur-core/tests/test_cashSender_usage.py

```diff
@@ -11,8 +11,9 @@ def test_cashSender_usage():
                 continue
             with open(os.path.join(root, file), "r") as auto:
                 data = auto.read()
-                expectedOccurences = 0
-                if (file == "Market.sol"):
-                    expectedOccurences = 1
-                notFound = data.count("cash.transfer") == expectedOccurences
-                assert notFound, "Contract %s has an unexpected cash.transfer in it" % file
\ No newline at end of file
+                notFound = data.count("cash.transfer") == 0
+                assert notFound, "Contract %s has an unexpected cash.transfer in it" % file
+                notFound = data.count("cash.balanceOf") == 0
+                assert notFound, "Contract %s has an unexpected cash.balanceOf in it" % file
+                notFound = data.count("cash.allowance") == 0
+                assert notFound, "Contract %s has an unexpected cash.allowance in it" % file
```

Comment:
There may be value in adding a method for `cash.approve` which will approve both DAI and vDAI.  Then we can add protection for `cash.approve` calls here as well.

We should also add a check for `cash.transferFrom` (though maybe that is counted in `cash.transfer` (my Python is weak)?

Reply:
yea cash.transferFrom is handled by this expression.

Reply:
for `cashApprove` thats a good point.